src/index.html

```diff
@@ -15,8 +15,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
Keep your attributes correctly formatted. The attributes of the link tag should start on a new line with 2-space indentation related to the tag. The closing bracket should be aligned with the opening bracket.